src/Chainweb/Pact/Backend/ChainwebPactDb.hs

```diff
@@ -829,7 +829,7 @@ deleteHistory bh = do
 
 initSchema :: BlockHandler SQLiteEnv ()
 initSchema = do
-    withSavepoint DbTransaction $ do
+    withSavepoint InitSchema $ do
```

Comment:
We have to double check that no code elsewhere makes assumptions about this savepoint name